src/main/java/net/earthcomputer/clientcommands/command/arguments/ClientItemPredicateArgumentType.java

```diff
@@ -51,11 +51,11 @@ public ClientItemPredicateArgument parse(StringReader reader) throws CommandSynt
         } else {
             Identifier tagId = itemReader.getId();
             return ctx -> {
-                @SuppressWarnings("ConstantConditions") Tag.Identified<Item> tag = (Tag.Identified<Item>) MinecraftClient.getInstance().getNetworkHandler().getTagManager().getItems().getTag(tagId);
+                @SuppressWarnings("ConstantConditions") Tag<Item> tag = MinecraftClient.getInstance().getNetworkHandler().getTagManager().getItems().getTag(tagId);
```

Comment:
Is `@SuppressWarnings("ConstantConditions")` still needed?

Reply:
> Is `@SuppressWarnings("ConstantConditions")` still needed?

In other words, do you get any warnings related to `ConstantConditions` around here if you remove the `@SuppressWarnings("ConstantConditions")` from that line? If not, the suppressor can be removed. If you do get such warnings, the suppressor should remain as is.

Reply:
Apparently it still has the effect of avoiding a null check